### PR TITLE
[MST-819] Convert onboarding profile API waffle flag to Django setting

### PR DESCRIPTION
**Pull request by bseverino**
**JIRA:**

[MST-819](https://openedx.atlassian.net/browse/MST-819)

**Pre-Merge Checklist:**

- [x] Updated the version number in `edx_proctoring/__init__.py` and `package.json` if these changes are to be released.
- [x] Described your changes in `CHANGELOG.rst`
- [ ] Confirmed Github reports all automated tests/checks are passing.
- [ ] Approved by at least one additional reviewer.

**Post-Merge:**

- [ ] Create a tag matching the new version number.

### CODE REVIEW

File: edx_proctoring/constants.py [hunk 1]
```diff
@@ -68,7 +68,11 @@
 
 PING_FAILURE_PASSTHROUGH_TEMPLATE = 'edx_proctoring.{}_ping_failure_passthrough'
 
-ONBOARDING_PROFILE_API = 'edx_proctoring.onboarding_profile_api'
+ONBOARDING_PROFILE_API = (
+    settings.PROCTORING_SETTINGS['USE_ONBOARDING_PROFILE_API'] if
+    'USE_ONBOARDING_PROFILE_API' in settings.PROCTORING_SETTINGS
+    else getattr(settings, 'USE_ONBOARDING_PROFILE_API', False)
```
Comment (alangsto) [inline edx_proctoring/constants.py:74]: Why do we have to check settings.PROCTORING_SETTINGS and settings?

Comment (bseverino) [reply edx_proctoring/constants.py:74]: I'm just aiming to be consistent with the rest of the settings in this file (I'm assuming it's for open edX considerations) but I guess it technically isn't necessary

**Review by schenedx — APPROVED** [review]
👍

File: edx_proctoring/constants.py (hunk 1 above)
Comment (alangsto) [reply edx_proctoring/constants.py:74]: Got it that makes sense!

**Review by alangsto — APPROVED** [review]

**Pull request merged**